janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -391,6 +392,11 @@ public ElasticSearchIndex(Configuration config) throws BackendException {
         setupStoredScripts();
     }
 
+    public static String determineIndexName(Configuration config) {
+        if ((!config.has(INDEX_NAME) && (config.has(GRAPH_NAME)))) return config.get(GRAPH_NAME);
```

Comment:
_nitpick_
Looks like too many redundant braces. Could be simplified to:
```
if (!config.has(INDEX_NAME) && config.has(GRAPH_NAME)) return config.get(GRAPH_NAME);
return config.get(INDEX_NAME);
```
Or even using ternary operator: 
```
return !config.has(INDEX_NAME) && config.has(GRAPH_NAME) ? config.get(GRAPH_NAME) : config.get(INDEX_NAME);
```

Reply:
Yes, you are right. I think we just copied this one from the CQL backend and we should fix it there as well.